t/op/ver.t

```diff
@@ -243,7 +243,9 @@ ok( exists $h{chr(65).chr(66).chr(67)}, "v-stringness is engaged for X.Y.Z" );
 
 {
     local $|;
+    no warnings 'numeric';
     $| = v0;
+    use warnings 'numeric';
```

Comment:
`{ no warnings; $| = v0 }`

Reply:
Okay, but my approach has been to try to make the warnings turned off as specific as possible.  Same thing for strictures, i.e., I don't say `no strict;` when `no strict 'vars';` will suffice.

Reply:
Addressed.